proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -93,7 +93,7 @@ message WorkflowExecutionInfo {
     int64 last_first_event_id = 18;
     int64 last_completed_workflow_task_started_event_id = 19;
     // Deprecated. use `WorkflowExecutionState.start_time`
-    google.protobuf.Timestamp start_time = 20;
+    reserved 20;
```

Comment:
We can't really remove this. Need to supported workflows started by old servers as well. Also need to support rollback case.

I think the best we can do is: 
1. add a util method in mutable state for getting StartTime. The implementation needs to fallback to executionInfo.StartTime if needed. Alternatively, populate executionState.StartTime based on executionInfo.StartTime in NewMutableStateFromDB.
2. Do dual write in next release. In release X+1 we can write only to the one in executionState.

Reply:
hm. this mean that previous code have a problem...
I will update the PR.